src/test/java/gov/noaa/pfel/erddap/handler/TopLevelHandlerTests.java

```diff
@@ -0,0 +1,61 @@
+package gov.noaa.pfel.erddap.handler;
+
+import gov.noaa.pfel.erddap.handlers.TopLevelHandler;
+import gov.noaa.pfel.erddap.util.EDStatic;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.xml.sax.SAXException;
+import testDataset.Initialization;
+
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.parsers.SAXParser;
+import javax.xml.parsers.SAXParserFactory;
+import java.io.IOException;
+import java.io.InputStream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TopLevelHandlerTests {
+
+    private static SAXParserFactory factory;
+    private static SAXParser saxParser;
+    private static InputStream inputStream;
+
+    @BeforeAll
+    static void initAll() throws ParserConfigurationException, SAXException {
+        Initialization.edStatic();
+
+        factory = SAXParserFactory.newInstance();
+        factory.setNamespaceAware(true);
+        factory.setXIncludeAware(true);
+        saxParser = factory.newSAXParser();
+    }
+
+    @BeforeEach
+    void init() {
+        inputStream = TopLevelHandlerTests.class.getResourceAsStream("/datasets/topLevelHandlerTest.xml");
+        if (inputStream == null) {
+            throw new IllegalArgumentException("File not found: /datasets/topLevelHandlerTest.xml");
+        }
+    }
+
```

Comment:
Can we add a test for user because that is handled so differently?

Reply:
I actually did try to add one for user. But could not think what exactly to test since no EDStatic attribute was being updated for the user tag

Reply:
You could check that the tUserHashMap has the expected values.